### PR TITLE
make quickplot use scaling from solutions

### PR DESCRIPTION
**Pull request by TomTranter**
# Description

Quickplot now uses timescale and lengthscale evals from the solution instead of evaluating them.

Fixes # 1233

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/master/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 -   Raise error if the boundary condition at the origin in a spherical domain is other than no-flux ([#1175](https://github.com/pybamm-team/PyBaMM/pull/1175))
 -   Fix boundary conditions at r = 0 for Creating Models notebooks ([#1173](https://github.com/pybamm-team/PyBaMM/pull/1173))
 -   Make sure simulation solves when evaluated timescale is a function of an input parameter ([#1218](https://github.com/pybamm-team/PyBaMM/pull/1218))
```
Comment (brosaplanella) [inline CHANGELOG.md:29]: Can you move these two lines of CHANGELOG so they are in descending order of PR?

**Review by brosaplanella — APPROVED** [review]
Looks good, just the minor point on the CHANGELOG. Thanks Tom!

File: CHANGELOG.md (hunk 1 above)
Comment (TomTranter) [reply CHANGELOG.md]: Done

**Comment by TomTranter** [comment]
@tinosulzer @rtimms some of the lines I changed don't actually seem to be used for anything. self.length_scales is defined but as far as I can tell never used anywhere. I commented them out and ran tests locally and they all passed. I also ran a 1+1 script and plotting the heat maps works ok without these lines. It does fail when the height is an input though. Not sure if worth fixing as I can't think of a scenario when  you'd want to have height as input when doing 1+1. Similarly I guess 2+1 will not like when height and width are inputs. I could add some more informative error messages

**Comment by rtimms** [comment]
yeah it looks like the length scale stuff gets handled in processed variable. and I agree it doesn't make sense for height to be an input in 1+1D as you're changing the geometry. maybe in `battery_geometry` we should raise an error if any of the parameters are inputs since you can change the geometry on the fly
